Makefile

```diff
@@ -849,9 +849,12 @@ update-testsuite:
 
 lint-packages: RUSTFLAGS += -D dead-code -D nonstandard-style -D unused-imports -D unused-mut -D unused-variables -D unused-unsafe -D unreachable-patterns -D bad-style -D improper-ctypes -D unused-allocation -D unused-comparisons -D while-true -D unconditional-recursion -D bare-trait-objects -D function_item_references # TODO: add `-D missing-docs`
 lint-packages:
-	RUSTFLAGS="${RUSTFLAGS}" cargo clippy --all --exclude wasmer-cli -- -D clippy::all
+	RUSTFLAGS="${RUSTFLAGS}" cargo clippy --all --exclude wasmer-cli --exclude wasmer-compiler-llvm $(compiler_features) -- -D clippy::all
```

Comment:
This should only be excluded if LLVM is disabled, not unconditionally.

Reply:
Done in https://github.com/wasmerio/wasmer/pull/3499/commits/c83b9e0455d1a906ba105258e6760a177d2816d9